docs/setup/cookbook/go-live.md

```
@@ -83,6 +83,9 @@ Going live is staged. The pilot starts with a small, internal go-live to confirm
 * Authentication success rate at the mock relying party portal.
 * Incident log --- what happened, what was done, what is still open.
 
+![](../../.gitbook/assets/fpg-figure-8.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)